README.md

```diff
@@ -61,6 +61,8 @@ If you wish to use [Paket](https://fsprojects.github.io/Paket/) for your depende
 dotnet new giraffe --Paket
 ```
 
+The template uses HTTPS redirection when run in production which is the default unless explicitly overrided. If you don't have a certificate configured for HTTPS, be sure to set `ASPNETCORE_ENVIRONMENT=Development`.
```

Comment:
`overrided` should be `overridden`, otherwise looks good :)

Maybe we could add this below as well:

```
In order to test production mode during development you can generate a self signed certificate using this guide:
https://docs.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide
```